test/modules/graphstorage.test.js

```diff
@@ -126,8 +153,12 @@ describe('GraphStorage module', () => {
     });
 
     after('drop myGraphStorage db', async () => {
-        systemDb = new Database();
-        systemDb.useBasicAuth('root', 'root');
-        await systemDb.dropDatabase(myDatabaseName);
+        if (selectedDatabase.database_system === 'arango_db') {
```

Comment:
same as above.

Graph storage needs to encapsulate all the information regarding the underlying database.